config/routes.rb

```diff
@@ -1,4 +1,14 @@
   Rails.application.routes.draw do
+  get 'courses/new'
```

Comment:
Are all these individual routes what you want? Looks like a great place to use `resources`